masterlist.yaml

```diff
@@ -2412,6 +2412,15 @@ plugins:
       - crc: 0x2EA38DC1
         util: 'FNVEdit v4.0.4c'
 
+  - name: 'Tree New Perks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/79005/'
+        name: 'TTW - Wasteland Supplements'
+    after: [ 'TTW Reputations.esm' ]
```

Comment:
Tested and default sorting already places `Tree New Perks.esp` after `TTW Reputations.esm`.
Even when both files have an ESM flag.
```suggestion
```